app/Plugin/CoreApi/Controller/ApiController.php

```diff
@@ -211,6 +212,31 @@ public function beforeFilter() {
     }
   }
 
+  /**
+   * Handle a Core API CO Person record create
+   *
+   * @since  COmanage Registry v4.1.0
+   */
+
+  public function create() {
+    try {
+      $ret = $this->CoreApi->createV1($this->cur_api['CoreApi']['co_id'],
```

Comment:
`createV1` doesn't appear to actually return anything? We need to convey the Identifier of the configured Identifier Type in the response.

Reply:
done